packages/connectivity/src/scp-cf/destination/destination-from-vcap.ts

```diff
@@ -83,6 +101,100 @@ export interface DestinationFromServiceBindingOptions {
   serviceBindingTransformFn?: ServiceBindingTransformFunction;
 }
 
+/**
+ * The application resource for which the token is requested for App-to-App communication.
+ * The token will only be usable to call the requested application.
+ * Either provide the app name (common case) or the provider client ID
+ * and tenant ID (optional).
+ */
+export type IasResource = Xor<
+  {
+    /**
+     * The name of the application resource.
+     */
+    name: string;
+  },
+  {
+    /**
+     * The client ID of the application resource.
+     */
+    providerClientId: string;
+    /**
+     * The tenant ID of the application resource (Optional).
+     */
+    providerTenantId?: string;
+  }
+>;
+
+/**
+ * Base options shared by all IAS authentication modes.
+ */
+interface IasOptionsBase {
+  /**
+   * The target URL of the destination that the IAS token is requested for.
+   * It is recommended to provide this for App-to-App communication (when resource parameter is used),
+   * otherwise the destination will point to the identity service URL from the service binding,
+   * instead of the actual target application. This function is not able to to infer
+   * the target application from the information available.
```

Comment:
```suggestion
   * instead of the actual target application. 
```
[pp] Sentence is verbose and can be omitted.